src/App/App.jsx

```diff
@@ -0,0 +1,16 @@
+import { Wrapper } from "./App.styles";
+import MainSection from "../components/MainSection/MainSection";
+import Navbar from "../components/Navbar/Navbar";
```

Comment:
jest trick (czasami używany, ale nie zawsze) na dodanie pliku index.js w folderze danego komponentu a w nim:
`export {default} from './Navbar`
dzięki temu ta ściezko uprości się i nie będzie 
`import Navbar from "../components/Navbar/Navbar";`
tylko 
`import Navbar from "../components/Navbar";`

Reply:
Ha, to ciekawe :D. Mimo, że nie kupuje mnie to rozwiązanie to zaimplementowałem bo pewnie warto zapamiętać.